### PR TITLE
feat: show unregistered RAS UI to admins

### PR DESCRIPTION
**Pull request by dkoo**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

When viewing RAS components as a logged-in editor or admin user, force the UI to show the "unregistered" state.

### How to test the changes in this Pull Request:

Instructions in https://github.com/Automattic/newspack-popups/pull/979.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

**Review by miguelpeixe — APPROVED** [review]
Non-blocking suggestion: disable the inputs with the admin email pre-filled to avoid confusion.

**Comment by dkoo** [comment]
Good idea! eff84ff

**Review by miguelpeixe — CHANGES_REQUESTED** [review]
Thanks! For readability, can we store the check in a variable?

```php
$is_admin_preview = method_exists( 'Newspack_Popups', 'is_user_admin' ) && \Newspack_Popups::is_user_admin();
```

**Comment by dkoo** [comment]
Good call! f821b82

**Review by miguelpeixe — APPROVED** [review]
Non-blocking

File: assets/blocks/reader-registration/index.php [hunk 1]
```diff
@@ -164,6 +165,8 @@ function render_block( $attrs, $content ) {
 		$success_login_markup = '<p class="has-text-align-center">' . $attrs['signedInLabel'] . '</p>';
 	}
 
+	$is_admin_preview = \method_exists( '\Newspack_Popups', 'is_user_admin' ) && \Newspack_Popups::is_user_admin();
```
[the remark this answers is not in the document]
Comment (dkoo) [reply assets/blocks/reader-registration/index.php]: Whoops, good call—moved the check earlier in 7d9f2b6.

> we don't usually apply the root namespace for PHP native functions

I think this applies to [another PR](https://github.com/Automattic/newspack-popups/pull/982), so I pushed the fix there.

[the remark this answers is not in the document]
Comment (miguelpeixe) [reply assets/blocks/reader-registration/index.php]: `method_exists()` as well

[the remark this answers is not in the document]
Comment (dkoo) [reply assets/blocks/reader-registration/index.php]: 🤦 

Is this what you mean? 4dd168f? Feel free to make a suggestion if I'm still not getting what you're saying!

[the remark this answers is not in the document]
Comment (miguelpeixe) [reply assets/blocks/reader-registration/index.php]: Yes! Thank you for revising! 🎉

**Review by miguelpeixe — APPROVED** [review]

**Pull request merged**